### PR TITLE
Move Unity into the common Getting Started

### PR DESCRIPTION
**Pull request by PeloWriter**
This PR stages the Unity content for further development, and moves Unity Lite into its own file (unity-lite.mdx). Note that the images used in the Config and Verify steps aren't currently displaying on my local, but I'm pushing this to GitHub as I _think_ it's a caching issue, so the Vercel preview should work. 

Please don't merge until the image issue is properly resolved.

### CODE REVIEW

File: src/includes/getting-started-config/unity.mdx [hunk 1]
```diff
@@ -1,6 +1,19 @@
-You can attach Sentry to a Game Object and initialize it with the DSN programatically:
+Currently, you can only configure Sentry using the Unity tools window.
 
-```csharp
-var sentry = gameObject.AddComponent<SentrySdk>();
-sentry.Dsn = "___PUBLIC_DSN___";
-```
+Open `Tools` -> `Sentry`.
```
Comment (bruno-garcia) [inline src/includes/getting-started-config/unity.mdx:3]: The message that comes from the common doesn't make much sense here. The config is done in the editor before building/running the game. So this doesn't make much sense:

image

Comment (PeloWriter) [reply src/includes/getting-started-config/unity.mdx:3]: I agree, will set it as a `PlatformSection` statement

File: src/platforms/unity/unity-lite.mdx [hunk 2]
```diff
@@ -20,23 +20,36 @@ This is a stable but lightweight SDK which doesn't support all of Sentry's featu
...
+```
 
 ## Configure
 
 Configuration should happen as early as possible in your application's lifecycle.
 
-<PlatformContent includePath="getting-started-config" />
+You can attach Sentry to a Game Object and initialize it with the DSN programatically:
+
+```C#
```
Comment (bruno-garcia) [inline src/platforms/unity/unity-lite.mdx:35]: To show the right highlighting:

````suggestion
```csharp
````

File: src/platforms/unity/unity-lite.mdx [hunk 3]
```diff
@@ -20,23 +20,36 @@ This is a stable but lightweight SDK which doesn't support all of Sentry's featu
...
+var sentry = gameObject.AddComponent<SentrySdk>();
+sentry.Dsn = "https://examplePublicKey@o0.ingest.sentry.io/0";
```
Comment (bruno-garcia) [inline src/platforms/unity/unity-lite.mdx:37]:
```suggestion
sentry.Dsn = "___PUBLIC_DSN___";
```

**Review by bruno-garcia — APPROVED** [review]
Thanks!

**Pull request merged**